.ci/jobs/pr-gke-full.yml

```diff
@@ -0,0 +1,30 @@
+---
+- job:
+    description: Build a PR version of ECK and run full suite of E2E tests
+    name: cloud-on-k8s-pr-full
+    project-type: pipeline
+    pipeline-scm:
+      scm:
+        - git:
+            url: https://github.com/elastic/cloud-on-k8s
+            branches:
+              - ${sha1}
+            credentials-id: 'f6c7695a-671e-4f4f-a331-acdce44ff9ba'
+            refspec: '+refs/pull/${ghprbPullId}/*:refs/remotes/origin/pr/${ghprbPullId}/*'
+      # Jenkinsfile shared with the release build job
+      script-path: .ci/pipelines/build.Jenkinsfile
```

Comment:
Maybe it should be https://github.com/elastic/cloud-on-k8s/blob/master/.ci/pipelines/e2e-tests-master-gke.Jenkinsfile? `.ci/pipelines/build.Jenkinsfile` seems to be pipeline for release